# Translations template for invenio-rdm-records.
# Copyright (C) 2022 CERN
# This file is distributed under the same license as the invenio-rdm-records
# project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2022.
# 
# Translators:
# Hermann Schranzhofer <hermann.schranzhofer@tugraz.at>, 2021
# Alizee Pace <alizee.pace@gmail.com>, 2021
# Tibor Simko <tibor.simko@cern.ch>, 2021
# Mojib Wali <mojib.wali@tugraz.at>, 2021
# chriz_uniba <christina.zeller@uni-bamberg.de>, 2021
# Alexander Gruber <alexander.gruber@tugraz.at>, 2021
# David Eckhard <dabombeeer@gmail.com>, 2022
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: invenio-rdm-records 0.34.4\n"
"Report-Msgid-Bugs-To: info@inveniosoftware.org\n"
"POT-Creation-Date: 2022-02-14 11:56+0100\n"
"PO-Revision-Date: 2021-07-12 14:31+0000\n"
"Last-Translator: David Eckhard <dabombeeer@gmail.com>, 2022\n"
"Language-Team: German (https://www.transifex.com/inveniosoftware/teams/23537/de/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 2.9.1\n"
"Language: de\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: invenio_rdm_records/config.py:183 invenio_rdm_records/config.py:440
#: invenio_rdm_records/config.py:449 invenio_rdm_records/config.py:470
#: tests/conftest.py:244 tests/conftest.py:253
msgid "DOI"
msgstr "DOI"

#: invenio_rdm_records/config.py:452 tests/conftest.py:256
msgid "OAI ID"
msgstr "OAI ID"

#: invenio_rdm_records/config.py:144
msgid "ORCID"
msgstr "ORCID"

#: invenio_rdm_records/config.py:149 invenio_rdm_records/config.py:268
msgid "ISNI"
msgstr "ISNI"

#: invenio_rdm_records/config.py:154
msgid "GND"
msgstr "GND"

#: invenio_rdm_records/config.py:159
msgid "ROR"
msgstr "ROR"

#: invenio_rdm_records/config.py:168
msgid "ARK"
msgstr "ARK"

#: invenio_rdm_records/config.py:173
msgid "arXiv"
msgstr "arXiv"

#: invenio_rdm_records/config.py:178
msgid "Bibcode"
msgstr "Bibcode"

#: invenio_rdm_records/config.py:188
msgid "EAN13"
msgstr "EAN13"

#: invenio_rdm_records/config.py:193
msgid "EISSN"
msgstr "EISSN"

#: invenio_rdm_records/config.py:198
msgid "Handle"
msgstr "Handle"

#: invenio_rdm_records/config.py:203
msgid "IGSN"
msgstr "IGSN"

#: invenio_rdm_records/config.py:208
msgid "ISBN"
msgstr "ISBN"

#: invenio_rdm_records/config.py:213
msgid "ISSN"
msgstr "ISSN"

#: invenio_rdm_records/config.py:218
msgid "ISTC"
msgstr "ISTC"

#: invenio_rdm_records/config.py:223
msgid "LISSN"
msgstr "LISSN"

#: invenio_rdm_records/config.py:228
msgid "LSID"
msgstr "LSID"

#: invenio_rdm_records/config.py:233
msgid "PMID"
msgstr "PMID"

#: invenio_rdm_records/config.py:238
msgid "PURL"
msgstr "PURL"

#: invenio_rdm_records/config.py:243
msgid "UPC"
msgstr "UPC"

#: invenio_rdm_records/config.py:248
msgid "URL"
msgstr "URL"

#: invenio_rdm_records/config.py:253
msgid "URN"
msgstr "URN"

#: invenio_rdm_records/config.py:258
msgid "W3ID"
msgstr "W3ID"

#: invenio_rdm_records/config.py:272
msgid "GRID"
msgstr "GRID"

#: invenio_rdm_records/config.py:276
msgid "Crossref Funder ID"
msgstr "Crossref Funder ID"

#: invenio_rdm_records/config.py:280
msgid "Other"
msgstr "Andere"

#: invenio_rdm_records/config.py:286
msgid "Wikidata"
msgstr "Wikidata"

#: invenio_rdm_records/config.py:290
msgid "GeoNames"
msgstr "GeoNames"

#: invenio_rdm_records/config.py:360
msgid "Best match"
msgstr "Höchste Übereinstimmung"

#: invenio_rdm_records/config.py:364
msgid "Newest"
msgstr "Neuester"

#: invenio_rdm_records/config.py:368
msgid "Oldest"
msgstr "Ältester"

#: invenio_rdm_records/config.py:372
msgid "Version"
msgstr "Version"

#: invenio_rdm_records/config.py:376
msgid "Recently updated"
msgstr "Kürzlich aktualisiert"

#: invenio_rdm_records/config.py:380
msgid "Least recently updated"
msgstr "Am wenigsten aktuell"

#: invenio_rdm_records/config.py:477
msgid "OAI"
msgstr "OAI"

#: invenio_rdm_records/ext.py:56
msgid "Your shared link has expired."
msgstr "Ihr freigegebener Link ist abgelaufen."

#: invenio_rdm_records/requests/community_submission.py:140
msgid "Community submission"
msgstr "Community Einreichung"

#: invenio_rdm_records/resources/config.py:93
msgid "Citation string style not found."
msgstr "Zitierungsstil nicht gefunden."

#: invenio_rdm_records/resources/config.py:99
msgid "Review for draft not found"
msgstr "Rezension für Entwurf nicht gefunden"

#: invenio_rdm_records/resources/serializers/datacite/schema.py:275
msgid "Invalid publication date value."
msgstr "Ungültiger Wert für das Veröffentlichungsdatum."

#: invenio_rdm_records/resources/serializers/ui/fields.py:35
#: invenio_rdm_records/services/facets.py:24
msgid "Open"
msgstr "Offen"

#: invenio_rdm_records/resources/serializers/ui/fields.py:36
#: invenio_rdm_records/services/facets.py:25
msgid "Embargoed"
msgstr "Gesperrt"

#: invenio_rdm_records/resources/serializers/ui/fields.py:37
#: invenio_rdm_records/services/facets.py:26
msgid "Restricted"
msgstr "Eingeschränkt"

#: invenio_rdm_records/resources/serializers/ui/fields.py:38
#: invenio_rdm_records/services/facets.py:27
msgid "Metadata-only"
msgstr "Nur Metadaten"

#: invenio_rdm_records/resources/serializers/ui/fields.py:65
msgid "The record and files are publicly accessible."
msgstr "Der Eintrag und die Dateien sind öffentlich zugänglich."

#: invenio_rdm_records/resources/serializers/ui/fields.py:67
msgid "No files are available for this record."
msgstr "Für diesen Eintrag sind keine Dateien verfügbar."

#: invenio_rdm_records/resources/serializers/ui/fields.py:74
#, python-format
msgid "The record and files will be made publicly available on %(date)s."
msgstr ""
"Der Eintrag und die Dateien werden am %(date)s öffentlich zugänglich gemacht"
" ."

#: invenio_rdm_records/resources/serializers/ui/fields.py:77
msgid "The record and files are restricted to users with access."
msgstr "Der Eintrag und die Dateien sind auf Benutzer mit Zugriff beschränkt."

#: invenio_rdm_records/resources/serializers/ui/fields.py:83
#, python-format
msgid "The record will be made publicly available on %(date)s."
msgstr "Der Eintrag wird am %(date)s öffentlich zugänglich gemacht. "

#: invenio_rdm_records/resources/serializers/ui/fields.py:86
msgid "The record is restricted to users with access."
msgstr "Der Eintrag ist auf Benutzer mit Zugriff beschränkt."

#: invenio_rdm_records/resources/serializers/ui/fields.py:91
#, python-format
msgid "The files will be made publicly available on %(date)s."
msgstr "Die Dateien werden am %(date)s öffentlich zugänglich gemacht am."

#: invenio_rdm_records/resources/serializers/ui/fields.py:94
msgid ""
"The record is publicly accessible, but files are restricted to users with "
"access."
msgstr ""
"Der Eintrag ist öffentlich zugänglich, aber die Dateien sind auf Benutzer "
"mit Zugriff beschränkt."

#: invenio_rdm_records/services/errors.py:36
msgid "Review not found."
msgstr "Rezension nicht gefunden."

#: invenio_rdm_records/services/facets.py:22
msgid "Access status"
msgstr "Zugriffsstatus"

#: invenio_rdm_records/services/facets.py:34
msgid "Status"
msgstr "Status"

#: invenio_rdm_records/services/facets.py:35
msgid "Published"
msgstr "Veröffentlicht"

#: invenio_rdm_records/services/facets.py:35
msgid "Unpublished"
msgstr "Unveröffentlicht"

#: invenio_rdm_records/services/facets.py:41
msgid "Languages"
msgstr "Sprachen"

#: invenio_rdm_records/services/facets.py:50
msgid "Resource types"
msgstr "Arten von Ressourcen"

#: invenio_rdm_records/services/facets.py:58
#: invenio_rdm_records/services/facets.py:64
msgid "Subjects"
msgstr "Themen"

#: invenio_rdm_records/services/components/review.py:37
msgid ""
"You cannot delete a draft with an open review. Please cancel the review "
"first."
msgstr ""
"Ein Entwurf mit einer ausstehenden Rezension kann nicht gelöscht werden. "
"Bitte brechen Sie die Rezension zuerst ab."

#: invenio_rdm_records/services/pids/manager.py:81
msgid "Invalid {scheme}"
msgstr "Ungültiger {scheme}"

#: invenio_rdm_records/services/pids/manager.py:136
msgid "A PID already exists for type {scheme}"
msgstr "Es existiert bereits ein PID vom Typ {scheme}"

#: invenio_rdm_records/services/pids/manager.py:142
msgid "External identifier value is required."
msgstr "Wert für externen Kennzeichner ist erforderlich."

#: invenio_rdm_records/services/pids/manager.py:180
#: invenio_rdm_records/services/pids/manager.py:209
msgid "PID not found for type {scheme}"
msgstr "PID vom Typ {scheme} nicht gefunden"

#: invenio_rdm_records/services/pids/manager.py:226
msgid "Cannot discard a reserved or registered persistent identifier."
msgstr ""
"Eine reservierte oder registrierte permanente Kennung kann nicht verworfen "
"werden."

#: invenio_rdm_records/services/pids/providers/base.py:154
msgid "{pid_type}:{identifier} already exists."
msgstr "{pid_type}:{identifier} existiert bereits."

#: invenio_rdm_records/services/pids/providers/external.py:41
msgid "The prefix '{prefix}' is administrated locally."
msgstr "Präfix \"{prefix}\" wird lokal verwaltet."

#: invenio_rdm_records/services/pids/providers/external.py:79
msgid "Missing {scheme} for required field."
msgstr "Fehlende {scheme} für Pflichtfeld."

#: invenio_rdm_records/services/review/service.py:48
msgid "A review already exists for this record"
msgstr "Für diesen Eintrag existiert bereits eine Rezension"

#: invenio_rdm_records/services/review/service.py:52
msgid "You cannot create a review for an already published record."
msgstr ""
"Für einen bereits veröffentlichten Eintrag kann keine Rezension erstellt "
"werden."

#: invenio_rdm_records/services/review/service.py:60
msgid "Invalid review type."
msgstr "Ungültiger Rezensions-Typ."

#: invenio_rdm_records/services/review/service.py:119
msgid ""
"You cannot delete a review for a draft that has already been published."
msgstr ""
"Eine Rezension für einen bereits veröffentlichten Entwurf kann nicht "
"gelöscht werden."

#: invenio_rdm_records/services/review/service.py:127
msgid "An open review cannot be deleted."
msgstr "Eine offene Rezension kann nicht gelöscht werden."

#: invenio_rdm_records/services/schemas/__init__.py:31
msgid "Invalid persistent identifier scheme."
msgstr "Ungültiges Schema für permanente Kennung."

#: invenio_rdm_records/services/schemas/access.py:40
msgid "Embargo end date must be set to a future date."
msgstr ""
"Das Enddatum für die Sperre muss auf ein zukünftiges Datum gesetzt werden."

#: invenio_rdm_records/services/schemas/access.py:49
msgid "Embargo end date must be unset or in the past."
msgstr ""
"Das Enddatum für die Sperre muss entweder nicht gesetzt sein oder in der "
"Vergangenheit liegen."

#: invenio_rdm_records/services/schemas/access.py:66
msgid "'{field_name}' must be either 'public' or 'restricted'"
msgstr "'{field_name}' muss entweder 'öffentlich' oder 'beschränkt' sein"

#: invenio_rdm_records/services/schemas/metadata.py:60
msgid "Only one value is accepted."
msgstr "Es wird nur ein Wert akzeptiert."

#: invenio_rdm_records/services/schemas/metadata.py:62
msgid "Not a valid locale."
msgstr "Keine gültiges Gebietsschema."

#: invenio_rdm_records/services/schemas/metadata.py:85
msgid "An existing id or a free text name must be present"
msgstr "Es muss eine vorhandene ID oder ein Freitextname vorhanden sein"

#: invenio_rdm_records/services/schemas/metadata.py:109
msgid "An existing id or a free text subject must be present"
msgstr "Eine vorhandene ID oder ein Freitext-Betreff muss vorhanden sein"

#: invenio_rdm_records/services/schemas/metadata.py:126
#: invenio_rdm_records/services/schemas/metadata.py:132
msgid "Invalid value. Choose one of {NAMES}."
msgstr "Ungültiger Wert. Wählen Sie einen von {NAMEN}."

#: invenio_rdm_records/services/schemas/metadata.py:154
msgid "Family name must be filled."
msgstr "Der Familienname muss ausgefüllt werden."

#: invenio_rdm_records/services/schemas/metadata.py:161
#: invenio_rdm_records/services/schemas/metadata.py:319
msgid "Name cannot be blank."
msgstr "Name kann nicht leer sein."

#: invenio_rdm_records/services/schemas/metadata.py:236
#: invenio_rdm_records/services/schemas/metadata.py:249
msgid "Not a valid URL."
msgstr "Keine gültige URL."

#: invenio_rdm_records/services/schemas/metadata.py:270
msgid "An existing id or a free text title must be present"
msgstr "Es muss eine vorhandene ID oder ein Freitextname vorhanden sein"

#: invenio_rdm_records/services/schemas/metadata.py:275
msgid ""
"Only an existing id or free text title/description/link is accepted, but not"
" both cases at the same time"
msgstr ""
"Entweder eine vorhandene ID oder Freitext Titel/Beschreibung/Link wird "
"akzeptiert, jedoch nicht beides zusammen."

#: invenio_rdm_records/services/schemas/metadata.py:330
msgid "Title cannot be blank."
msgstr "Titel kann nicht leer sein."

#: invenio_rdm_records/services/schemas/metadata.py:334
msgid "Number cannot be blank."
msgstr "Zahl kann nicht leer sein."

#: invenio_rdm_records/services/schemas/metadata.py:353
msgid "At least award or funder should be present."
msgstr "Zumindest Auftraggeber oder Geldgeber sollte vorhanden sein."

#: invenio_rdm_records/services/schemas/metadata.py:391
msgid ""
"At least one of ['geometry', 'place',                 'identifiers', "
"'description'] shold be present."
msgstr ""
"Mindestens eines von ['geometry', 'place', 'identifiers', 'description'] "
"muss vorhanden sein."

#: invenio_rdm_records/services/schemas/metadata.py:411
msgid "Missing data for required field."
msgstr "Fehlende Daten für Pflichtfeld."

#: invenio_rdm_records/services/schemas/metadata.py:429
msgid "Size cannot be a blank string."
msgstr "Größe kann keine leere Zeichenkette sein."

#: invenio_rdm_records/services/schemas/metadata.py:431
msgid "Format cannot be a blank string."
msgstr "Format kann keine leere Zeichenkette sein."

#: invenio_rdm_records/services/secret_links/service.py:71
#: invenio_rdm_records/services/secret_links/service.py:103
msgid "Expiration date must be set to the future"
msgstr "Ablaufdatum muss in der Zukunft liegen"

#: invenio_rdm_records/services/secret_links/service.py:97
msgid "Cannot postpone expiration of links"
msgstr "Ablauf von Links kann nicht verschoben werden"

#: invenio_rdm_records/services/secret_links/service.py:133
msgid "An access permission level is required"
msgstr "Eine Zugriffsberechtigungsstufe ist erforderlich"

#: invenio_rdm_records/services/secret_links/service.py:146
msgid "Invalid access permission level."
msgstr "Ungültige Zugriffsberechtigungsstufe."
